jsonrpc/server.go

```diff
@@ -254,19 +254,13 @@ func (s *Server) handleRequest(req *request) (*response, error) {
 
 	calledMethod, found := s.methods[req.Method]
 	if !found {
-		res.Error = &Error{
-			Code:    MethodNotFound,
-			Message: "method not found",
-		}
+		res.Error = rpcErr(MethodNotFound, "method not found")
```

Comment:
I think we can just pass `nil` as `data` here, right? It looks redundant.

Reply:
should be `nil` not empty string

Reply:
Done